res/values/strings.xml

```diff
@@ -285,24 +285,24 @@
 
     <string name="videochat">Video Chat</string>
     <string name="videochat_invite_user_to_videochat">Invite %1$s to a video chat?</string>
-    <string name="videochat_invite_user_hint">This requires a compatible app or a compatible browser on both ends.</string>
+    <string name="videochat_invite_user_hint">This requires a compatible app or browser on both ends.</string>
     <string name="videochat_contact_invited_hint">%1$s invited to a video chat.</string>
-    <string name="videochat_you_invited_hint">You invited to a video chat.</string>
+    <string name="videochat_you_invited_hint">You were invited to a video chat.</string>
```

Comment:
mmmh... I think this is not right, the string is shown to yourself like "you did invite to a video chat" on the receiver side it is "Foo invited to a video chat" look the previous string